src/Visualization/Python/PlotTrajectories.py

```diff
@@ -0,0 +1,155 @@
+@click.option(
+    "-o",
+    "--output",
+    type=str,
+    help=(
+        'Output filename. Choose ".pdf" '
+        'or ".png". If not provided, the plot will be displayed in a '
+        "graphic window."
+    ),
+)
+@click.option(
+    "--force", is_flag=True, help="Force overwriting existing output file."
+)
+def plot_trajectory_command(simulation_name, output, force):
+    """Plot trajectories from inspiral simulation
+
+    Concatenates partial trajectories from each Segment and plots full
+    trajectories.
+    """
+    reductionfiles = select_reduction_files("./", simulation_name)
+    AhA, AhB = import_A_and_B(reductionfiles)
+    if AhA is not None and AhB is not None:
+        plot_trajectory(AhA, AhB, simulation_name, output, force)
+    else:
+        click.echo("Error: Unable to import data.")
+
+
+def select_reduction_files(dir_path, simname):
+    reduction_files_list = glob.glob(
+        os.path.join(dir_path, "**", "BbhReductions.h5"), recursive=True
+    )
+    selected_dir_list = [
+        path for path in reduction_files_list if "Segment" in path
+    ]
+    selected_file_list = [file for file in selected_dir_list if simname in file]
+    return selected_file_list
+
+
+def import_A_and_B(filenames):
+    A_data = []
+    B_data = []
+
+    for filename in filenames:
+        try:
+            with h5py.File(filename, "r") as file:
+                if "ApparentHorizons/ControlSystemAhA_Centers.dat" in file:
+                    A_data.append(
+                        np.array(
+                            file[
+                                "ApparentHorizons/ControlSystemAhA_Centers.dat"
+                            ][:]
+                        )
+                    )
+                if "ApparentHorizons/ControlSystemAhB_Centers.dat" in file:
+                    B_data.append(
+                        np.array(
+                            file[
+                                "ApparentHorizons/ControlSystemAhB_Centers.dat"
+                            ][:]
+                        )
+                    )
+        except Exception as e:
+            click.echo(f"Error reading file '{filename}': {e}")
+            return None, None
+
+    if A_data and B_data:
+        A = np.concatenate(A_data)
+        B = np.concatenate(B_data)
+        min_length = min(len(A), len(B))
+        return A[:min_length], B[:min_length]
+    else:
+        return None, None
+
+
+def plot_trajectory(AhA, AhB, plotlabel="", output=None, force=False):
+    indexs = [4, 5]  # 4 -> x, 5 -> y
+
+    if output and os.path.exists(output) and not force:
+        click.echo(
+            f"Warning: Output file '{output}' already exists. "
+            "Use --force to overwrite."
+        )
+        return
+
+    plt.figure(figsize=(10, 5))
+    plt.subplot(1, 2, 1)
+    plt.plot(
+        [AhA[i][indexs[0]] for i in range(len(AhA))],
+        [AhA[i][indexs[1]] for i in range(len(AhA))],
+        label="AhA",
+        color="blue",
```

Comment:
`color="C0"` (that's the default and can be configured by the user)

Suggestion: I think it would be useful to fade the color from opacity 0 to 1 based on the time parameter, or maybe from some opacity < 1 to 1. That would give a sense of the time evolution. Maybe also plot `marker="."` at the very end of the trajectory to highlight the current position.